src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -346,6 +347,10 @@ export class ExposureNotificationService {
     }
   }
 
+  async clearExposedStatus() {
+    return this.finalize({type: ExposureStatusType.Monitoring});
```

Comment:
What is being returned here? `finalize` doesn't return a value.

Reply:
It return the promise this happens in `performExposureStatusUpdate` as well